### ISSUE TITLE
Remove virtual location from Mandatory fields section.Only Location is needed.

### ISSUE THREAD

**Issue by troughc** [culturecreates/footlight-app#1224]

**Comment by troughc** [comment]
Tested: Passed with one small request to fix:
- On the event form, virtual location was accessible on the right rather than always displayed in the form. Now it seems to always be displayed. Can you go back to the way it was?

**Comment by AbhishekPAnil** [comment]
> Tested: Passed with one small request to fix:
> - On the event form, virtual location was accessible on the right rather than always displayed in the form. Now it seems to always be displayed. Can you go back to the way it was? 
> - @AbhishekPAnil let's think about if there is a logic conflict of some sort. I say this because I see that location is a group of parameters like Contact has a group of paramters.
Yes I can change it to the old way of displaying it on right side.
You are right about the logical conflict.If contact is regarded as mandatory then any of the contact fields would be sufficient to satisfy the mandatory field requirement.Right ?

**Comment by troughc** [comment]
Yes, that is correct and I believe that this is how Contact currently works @AbhishekPAnil. Is that right? No rush. :)

**Comment by troughc** [comment]
@AbhishekPAnil I spoke too soon. the way Contact works is that the user can select a Contact field to be mandatory in Mandatory fields. Then all the Contact fields are displayed in the Events/Orgs forms. This is good and I am currently creating a feature enhancement request related to this. We can discuss Monday.

**Comment by AbhishekPAnil** [comment]
@troughc The changes are updated in the PR.

**Comment by troughc** [comment]
Tested: PASSED
